packages/blockchain-link/src/workers/blockbook/websocket.ts

```diff
@@ -256,6 +258,22 @@ export default class Socket extends EventEmitter {
         return this.send('estimateFee', payload);
     }
 
+    getCurrentFiatRates(currencies?: string[]) {
+        return this.send('getCurrentFiatRates', { currencies });
+    }
+
+    getAccountBalanceHistory(payload: AccountBalanceHistoryParams) {
+        return this.send('getBalanceHistory', payload);
+    }
+
+    getFiatRatesForTimestamps(timestamps: number[], currencies?: string[]) {
```

Comment:
unnecessary object > fn params > object conversion
use GetFiatRatesForTimestamps['payload'] object, it's already typed